src/leopard_em/analysis/pick_match_template_peaks.py

```diff
@@ -189,6 +190,7 @@ def extract_peaks_and_statistics(
         theta=best_theta[pos_y, pos_x],
         phi=best_phi[pos_y, pos_x],
         relative_defocus=best_defocus[pos_y, pos_x],
+        refined_relative_defocus=best_defocus[pos_y, pos_x],
```

Comment:
Where is this refined_relative_defocus being used? This is just a duplicate of the `relative_defocus` at this point